rest_docs/rest-catalog-open-api.yaml

```diff
@@ -163,7 +154,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/responses/IcebergErrorResponse'
+                $ref: '#/components/schemas/ErrorModel'
```

Comment:
This was what was causing the problem.

The `ErrorModel` is just the fields, without the outer wrapping JSON keyed by "error". We can update that in another pass if desired, but I wanted to keep the change set small to make this more digestible.